### ISSUE TITLE
Is this package going to be maintained?

### ISSUE THREAD

**Issue by mohamed-akef** [OWASP/rbac#88]
why no new commits in the package from 2015, I need to use it in large scale app and I'm afraid because of maintained.
any help?

**Comment by KnightYoshi** [comment]
It's an OWASP project and can be updated by anybody, which means it's not really going anywhere.

Apache's Log4PHP hasn't been updated in 4yrs, 
https://github.com/apache/logging-log4php

When something works as intended, bug free, they don't have to make regular updates to it.

**Comment by orinoco** [comment]
Of course, the fact that phprbac.net is down, and thus the docs, is not encouraging. You can use http://web.archive.org/web/20161104111344/http://phprbac.net in the meantime.

<img width="696" alt="phprbac net expired" src="https://cloud.githubusercontent.com/assets/3967161/23436014/42526144-fdc6-11e6-9403-25db36e4494f.png">

**Comment by abiusx** [comment]
I will try to bring the website back up.
